packages/cli/src/bin/pipcook-daemon.ts

```diff
@@ -36,13 +41,24 @@ async function start(): Promise<void> {
     if (message.event === 'ready') {
       daemon.disconnect();
       daemon.unref();
-      logger.success(`Pipcook service started on ${message.data.listen}. To open pipboard, please visit https://pipboard.vercel.app`);
+      let serverInfo;
+      // TODO(Feely): the listen field is deprecated, we will remove this
```

Comment:
When do we remove it?

Reply:
Maybe v2.0.0?

Reply:
This might not be a breaking change, right?

Reply:
Yes, if remove this field, the old version of pipcook-cli will not work as expect.